### PR TITLE
Add new remove-saas CLI

### PR DESCRIPTION
**Pull request by wallyworld**
## Description of change

juju remove-application is no longer used to remove a SAAS application.
There's now a remove-saas command, aliased to remove-consumed-application

## QA steps

consume an offer, see that it is in status under SAAS
`juju remove-saas <saas name>`
run status and check that it's gone

## Documentation changes

We'll need to document the new remove-saas command, and adjust doc for remove-application

### CODE REVIEW

**Comment by nskaggs** [comment]
@pmatulis new cli incoming.

**Comment by pmatulis** [comment]
Docs primed. Waiting on a new Juju to update the commands.md file.

**Comment by howbazaar** [comment]
[remove-saas command](https://trello.com/c/46xibY7q/2-remove-saas-command)

File: apiserver/params/params.go [hunk 1]
```diff
@@ -498,6 +498,12 @@ type DestroyApplicationParams struct {
 	DestroyStorage bool `json:"destroy-storage,omitempty"`
 }
 
+// DestroyConsumedApplicationsParams holds bulk parameters for the
+// Application.DestroyConsumedApplication call.
+type DestroyConsumedApplicationsParams struct {
+	ApplicationTags []string `json:"application-tags"`
```
Comment (axw) [inline apiserver/params/params.go]: please structure this like:

```go
type DestroyConsumedApplicationsParams struct {
    Applications []DestroyConsumedApplicationParams `json:"applications"`
}

type DestroyConsumedApplicationParams struct {
    ApplicationTag string `json:"application-tag"`
}
```

so we can add application-specific flags as necessary

**Comment by wallyworld** [comment]
$$merge$$

**Pull request merged**